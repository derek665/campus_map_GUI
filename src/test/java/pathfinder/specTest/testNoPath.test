# test correct output when no path is found

CreateGraph noPath

AddNode noPath A
AddNode noPath B

FindPath noPath A B