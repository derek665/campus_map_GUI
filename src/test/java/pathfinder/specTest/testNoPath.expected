# test correct output when no path is found

created graph noPath

added node A to noPath
added node B to noPath

no path found